Firmware/variants/MK3S-E3DREVO_HT_40W.h

```diff
@@ -0,0 +1,674 @@
+#ifndef CONFIGURATION_PRUSA_H
+#define CONFIGURATION_PRUSA_H
+
+#include <limits.h>
+#include "printers.h"
+/*------------------------------------
+ GENERAL SETTINGS
+ *------------------------------------*/
+
+// Printer revision
+#define PRINTER_TYPE PRINTER_MK3S
+#define PRINTER_NAME PRINTER_MK3S_NAME
+#define PRINTER_NAME_ALTERNATE PRINTER_MK3_NAME //the other similar printer to this.
+#define PRINTER_MMU_TYPE PRINTER_MK3S_MMU3
+#define PRINTER_MMU_NAME PRINTER_MK3S_MMU3_NAME
+#define FILAMENT_SIZE "1_75mm_MK3S"
+#define NOZZLE_TYPE "E3DREVO_HT_40W"
+
+// Printer name
+#define CUSTOM_MENDEL_NAME "Prusa i3 MK3S-RHT40"
```

Comment:
Name is to long it is limited to 17 chars at the moment. Also the PR contains a PT1000 thermistor and as there is a HT PT100 version available the naming should also be different.
```
01234567890123456789
Prusa MK3S-RHT1k          //for E3d Revo HT with PT1000
``` 
the other 3 options could be
01234567890123456789
Prusa MK3S-RHT1c          //for E3d Revo HT with PT100 
Prusa MK3S-RUHT1c       //for E3d Revo UHT with PT100
Prusa MK3S-RUHT1k       //for E3d Revo UHT with PT1000
```